web_src/js/modules/fomantic.js

```diff
@@ -22,6 +22,41 @@ export function initGiteaFomantic() {
     return escape(text, preserveHTML) + svg('octicon-x', 16, `${className.delete} icon`);
   };
 
+  // stand-in for removed transition module
+  $.fn.transition = function (arg) {
+    if (arg === 'is supported') return true;
+    if (arg === 'is animating') return false;
+    if (arg === 'is inward') return false;
+    if (arg === 'is outward') return false;
+    if (arg === 'stop all') return;
+
+    const isIn = arg?.animation?.endsWith(' in');
+    const isOut = arg?.animation?.endsWith(' out');
+
+    let ret;
+    if (arg === 'show' || isIn) {
+      arg?.onStart?.(this);
+      ret = this.each((_, el) => {
+        $(el).removeClass('hidden');
+        $(el).addClass('visible');
```

Comment:
I guess there's no way to use our own mechanism with `gt-hidden`, correct?

Reply:
No, they are different. Fomantic always use "hidden" internally.

Reply:
No, other fomantic code relies on these classes.